sim/hunter/item_sets_pve.go

```diff
@@ -191,3 +191,170 @@ var ItemSetGiantstalkerPursuit = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetDragonstalkerProwess = core.NewItemSet(core.ItemSet{
+	Name: "Dragonstalker's Prowess",
+	Bonuses: map[int32]core.ApplyEffect{
+		// Raptor Strike increases the damage done by your next other melee ability within 5 sec by 20%.
+		2: func(agent core.Agent) {
+			hunter := agent.(HunterAgent).GetHunter()
+
+			procAura := hunter.RegisterAura(core.Aura{
+				ActionID: core.ActionID{SpellID: 467329},
+				Label:    "S03 - Item - T2 - Hunter - Melee 2P Bonus",
+				Duration: time.Second * 5,
+				OnGain: func(aura *core.Aura, sim *core.Simulation) {
+					for _, spell := range hunter.MeleeSpells {
+						spell.DamageMultiplier *= 1.20
+					}
+				},
+				OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+					for _, spell := range hunter.MeleeSpells {
+						spell.DamageMultiplier /= 1.20
+					}
+				},
+				OnCastComplete: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell) {
+					if !spell.ProcMask.Matches(core.ProcMaskMeleeMHSpecial) {
+						return
+					}
+
+					aura.Deactivate(sim)
+					if spell.SpellCode == SpellCode_HunterRaptorStrike {
+						aura.Activate(sim)
+					}
+				},
```

Comment:
With the below aura won't this double proc it? It's minor but you could probably just return if the spell is raptor strike and never disable it since it'll be refreshed anyways

Reply:
Yeah the problem I had was that if raptor strike consumed the buff it wouldn't reactivate it so I manually added it here, I could change it to simply ignore raptor strike instead of doing this reactivation